packages/url_launcher/ios/Classes/UrlLauncherPlugin.m

```diff
@@ -33,19 +34,20 @@ - (instancetype)initWithUrl:url withFlutterResult:result {
 
 - (void)safariViewController:(SFSafariViewController *)controller
       didCompleteInitialLoad:(BOOL)didLoadSuccessfully API_AVAILABLE(ios(9.0)) {
-  if (didLoadSuccessfully) {
```

Comment:
I don't believe this should be erased. With this change, the `Future` on the dart side now doesn't finish until the `WebView` is closed by the User. Right now it finishes after the URL is loaded. 

To avoid breaking changes, you could keep this code and then set `flutterResult = null` if this method is called. Then in `safariViewControllerDidFinish` you would only use `flutterResult` if it isn't `null`.

Or you could add another parameter that checks if `safariViewController` was called. Either way should work.